cmake/CompilerWarnings.cmake

```diff
@@ -19,6 +19,9 @@ function(set_project_warnings project_name)
       /wd4458 # declaration of 'var' hides class member
       /wd4459 # declaration of 'var' hides global declaration
 
+      # Very common in WolfSSL
+      /wd4820 # 'n' bytes padding added after data member 'member'
```

Comment:
Can you check to see if this is needed anymore if we're dumping WOlfSSL?

Reply:
Removed